src/observable.js

```diff
@@ -114,6 +114,52 @@ extend(Observable.prototype, {
     return this._off(ANY, fn);
   },
 
+  subscribe(observer) {
+    const _this = this;
+    let closed = false;
+
+    if (typeof observer === 'function') {
+      observer = {
+        next: observer,
+        error: arguments[1],
```

Comment:
Yeah, in the [es-observable](https://github.com/zenparsing/es-observable) spec, under [Observable](https://github.com/zenparsing/es-observable#observable).